src/actors/cficaches.rs

```diff
@@ -201,6 +202,7 @@ impl CfiCacheActor {
     ) -> impl Future<Item = Arc<CfiCacheFile>, Error = Arc<CfiCacheError>> {
         let object = self
             .objects
+            .clone()
```

Comment:
I hope this is fast to clone…

Reply:
It's 4 `Arc`s so not too bad.  Also, if you looked into this this was already being cloned many times (still is) so this isn't that bad of a *change* compared to before.